server/src/test-data/fixtures.js

```diff
@@ -22,25 +22,37 @@ export const DEFAULT_GROUP = 'NSW SES';
 
 export const ORG_NAME = 'NSW SES';
 
-export const CAPABILITIES = [
-  'Road Crash Rescue',
-  'Vertical Rescue',
-  'Flood Rescue',
-  'Chainsaw',
-];
 
 export const TAGS = [
-  'NSW',
-  'SWR',
-  'ISR',
-  'SSR',
-  'KMA',
-  'PAR',
-  'HLS',
-  'WOL',
-  'AUB',
-  'HAW',
-  'HOL',
+  { name: 'NSW', type: 'orgStructure' },
```

Comment:
instead of copy paste why don't you make the code work for you?
```
const makeTags = (rawTags) => {
  const results = [];
  rawTags.each((tags, type) => {
    tags.each((tag) => {
      results.push({ name: tag, type });
    });
  });
  return results;
});

export const TAGS = makeTags({
  orgStructure: [
    'SWR',
    'ISR',
    ...
  ],
  capability: [
    'Storm and Water Damage - Ground',
    'Storm and Water Damage - Heights',
  ],
});
```

Reply:
Done. for the record when did it become so hard to walk over an object! you cant use .each on an object so the above didnt work. used...

```
const makeTags = (rawTags) => {
  const results = [];
  Object.entries(rawTags).forEach((entry) => {
    const [type, value] = entry;
    value.forEach((tag) => {
      results.push({ name: tag, type });
    });
  });
  return results;
};
```